partner_risk_insurance/views/res_partner_view.xml

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+    <record model="ir.ui.view" id="view_partner_form">
+        <field name="name">res.partner.form.risk_insurance</field>
+        <field name="model">res.partner</field>
+        <field name="type">form</field>
+        <field name="inherit_id" ref="partner_financial_risk.res_partner_view_risk" />
+        <field name="arch" type="xml">
+            <page name="financial_risk" position="inside">
+                <group name="credit_insurance" string="Credit Insurance">
+                    <group>
+                        <group>
+                            <field name="company_credit_limit" />
```

Comment:
Apply styles to avoid:
![seleccion_259](https://user-images.githubusercontent.com/7701001/40071999-b53ad422-5872-11e8-8696-3597549477c4.png)